pkgs/top-level/python-aliases.nix

```diff
@@ -59,6 +59,7 @@ mapAliases ({
   prompt_toolkit = prompt-toolkit;
   pylibgen = throw "pylibgen is unmaintained upstreamed, and removed from nixpkgs"; # added 2020-06-20
   pymssql = throw "pymssql has been abandoned upstream."; # added 2020-05-04
+  pysmu = libsmu; # added 2021-09-19
```

Comment:
We only add aliases when renaming packages.

```suggestion
```

Reply:
ok, removed alias, named the python package `pysmu` instead
(it seems to be the main way upstream refers to it and uses `from pysmu import *`, though annoyingly they also refer to the package as "libsmu with python bindings" and "libpysmu")